website/content/faq.md

```diff
@@ -4,3 +4,10 @@ description = "Frequently Asked Questions"
 weight = 4
 +++
 
+# Can I use Valence as a replacement for a Vanilla Minecraft Server?
+
+No.
+
+Valence is a Minecraft server *framework*. The goal of Valence is to provide a platform for developers to build custom Minecraft servers. **Valence, on it's own, is not a direct replacement for a Vanilla Minecraft server.** However, it should be flexible enough to build a Vanilla Minecraft server on top of it. You just have to implement the game logic yourself.
+
+Valence is best suited for building minigame servers that can handle large amounts of players. It intensionally implements the least amount of Vanilla Minecraft so that you can appropriately override and customize the logic.
```

Comment:
```suggestion
Valence is best suited for building minigame servers that can handle large amounts of players. It intentionally implements the least amount of Vanilla Minecraft so that you can appropriately override and customize the logic.
```

Reply:
Good catch. I'm more concerned that the typos ci check didn't pick it up though. @rj00a could this be an upstream bug?

Reply:
"Intensional" is an actual word, so probably not a bug.